index.js

```diff
@@ -12,7 +12,8 @@ try {
  * Element prototype.
  */
 
-var proto = Element.prototype;
+var Element = require('global-object').Element;
```

Comment:
Seemed more immediately comprehensible to use `global-object`. Happy to go with whatever the maintainers like in order to get this merged though.